.github/workflows/npm-publish-prerelease.yml

```diff
@@ -0,0 +1,20 @@
+name: npm Package
+
+on:
+  release:
+    types: [prereleased]
+
+jobs:
+  publish-npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14"
+          registry-url: "https://registry.npmjs.org"
+      - run: npm ci
+      - run: npm --no-git-tag-version version ${{ github.event.release.tag_name }}
+      - run: npm publish --access public --tag next
```

Comment:
only actual change is the `next` tag (this flow is triggered on `prereleased`)

Reply:
This is even better than the manual release I planned to do 👍